src/stock.lua

```diff
@@ -161,18 +161,25 @@ function stock.buy(metaname, kst) -- returns true if a return is required, or a
   local stack = 64 -- TODO: Get stack based on invItem.id, instead of hardcoding it.
   -- This means items such as Ender Pearls will entierly break the whole shop.
 
+  local mPushed = 0
```

Comment:
I prefer way more descriptive variable names. It would be great
if this was called `totalPushed` and `mOverflow` was called `underFlow` later on.

Reply:
also fixed in commit https://github.com/yourfriendoss/yfshop/pull/1/commits/d62f003c24c33140b1479ac15cf31d8a0475e214